enforcer-rules/src/test/java/org/apache/maven/enforcer/rules/files/TestRequireFilesDontExist.java

```diff
@@ -121,11 +124,11 @@ void testFileDoesNotExistSatisfyAny() throws EnforcerRuleException, IOException
         File f = File.createTempFile("junit", null, temporaryFolder);
         f.delete();
 
-        assertFalse(f.exists());
+        assumeTrue(f.exists());
```

Comment:
I don't get this one. The file is shall not be there (method name says and `f.delete`) but you assume it exists?

Reply:
fixed